connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala

```diff
@@ -298,10 +298,13 @@ object KustoDataSourceUtils {
 
   private[kusto] def getEngineUrlFromAliasIfNeeded(cluster: String): String = {
     if (cluster.startsWith(EnginePrefix)) {
-      val host = new URI(cluster).getHost
+      val uri = new URI(cluster)
+      val host = uri.getHost
```

Comment:
Open issue to add optional ingest cluster URL like in Kafka, but with a fallback here